docs/usage/configuration/config_documentation.md

```diff
@@ -3864,7 +3869,8 @@ instance_map:
 
 Experimental: When using workers you can define which workers should
 handle writing to streams such as event persistence and typing notifications.
-Any worker specified here must also be in the [`instance_map`](#instance_map).
+Any worker specified here must also be in the [`instance_map`](#instance_map), the 
+exception being the main process.
```

Comment:
to be honest I feel like this phrasing just adds more confusion, I'd probably settle towards the little innocent lie that you need to have all the workers, even main, defined in `instance_map`, even though we still have the backwards compat?

Reply:
This particular block was about `stream_writers`, I thought...?

Reply:
I guess I'm not sure I'm following 'the exception being the main process' here

Reply:
The exception is that the main process doesn't need to be anywhere on the `stream_writers` mapping, but everything else from the `instance_map` needs an entry here.

Reply:
Ok, I'm convinced. Fixed in [f6c24e4](https://github.com/matrix-org/synapse/pull/15491/commits/f6c24e49751b5f9790b0686b6a8d36c27745f8eb)